### DISCUSSION TITLE
[OP Stack Deployment] Changing the block gas limit

### DISCUSSION THREAD

**Discussion by fuzious** [ethereum-optimism/developers#515] [Deploying the OP Stack]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

Hi guys, I want to change the block gas limit to 600M , for testing purposes , I changed the genesis.json with 600M and rollup.json and restarted op-node but still the blocks being produced are of 30M only (the previous limit) .
I could not start op-geth with new config as with changed genesis.json it showed errors
Can you please help me with it?

So I restarted op-geth with the flag miner.gaslimit 600000000 but still I am getting the old limit 
1) The genesis.json in op-geth has 30M limit while in op-node folder i changed it to 600M 
2) rollup.json has 600M limit
3) I have restarted op-node with the same command 
4) Restarted op-geth with the additional flag of miner.gasLimit

Still I am seeing no changes, can someone help me with it please. Please note that the chain is managed by me with 4M blocks already published, here is the explorer link https://nolliescan.skatechain.org/

How to modify the `SystemConfig` contract which is already deployed, in case it is required.
Thanks

**Comment by opfocus** [comment]
Can this be changed? I've only heard of changes to the `Block gas target`.   https://docs.optimism.io/chain/differences#eip-1559-parameters     CC @sbvegan

**Reply by fuzious** [reply]
I thought that block gas limit could be set to any value but not sure how to do it for an already running chain. CC @sbvegan
